test/integration/enumerate_databases.test.ts

```diff
@@ -233,6 +233,6 @@ const testSuite = new UnifiedTestSuiteBuilder('listDatabases with comment option
   )
   .toJSON();
 
-describe('listDatabases w/ comment option', () => {
+describe('listDatabases with comment option', () => {
```

Comment:
non-blocking: Feel free to use the new `toMocha` method you added on `enumerate_databases` and `enumerate_indexes` if you'd like, but not necessary for the review!

Reply:
Done!